### ISSUE TITLE
Opendataval is slow on DataShaply.

### ISSUE THREAD

**Issue by TheaperDeng** [opendataval/opendataval#33]
### 🐛 Describe the bug

Hi @kevinfjiang Thank you for providing the library.

I found that `opendataval` is VERY SLOW on DataShapley with default parameters. 

```python
# DataShapley may need several days on A40
dataval = DataShapley()

# DataBanzhaf completed in several hours on A40
dataval = DataBanzhaf(num_models=10000)


dataval = dataval.input_model(openval_model)\
                 .input_metric(loss_func)\
                 .input_data(x_train, y_train, x_val, y_val)\
                 .train_data_values()
```

Here openval_model is just a MLP (2 layer) with 0.11M paramters. x_train and x_val are ~10K images from MNIST-10 dataset, and loss_func is just a cross-validation loss.

I want to understand if this is a normal case, and can I tune some hyperparameters to make the algorithm faster?

### Versions

1.3.0

### Operating System

Linux
